core/templates/dev/head/pages/exploration_player/conversation_skin_directive.html

```diff
@@ -383,7 +375,7 @@
     }
 
     .conversation-skin-tutor-card-container {
-      position: absolute;
+      box-shadow: 0 2px 4px rgba(0, 0, 0, 0.14), 0 2px 4px rgba(0, 0, 0, 0.23);
```

Comment:
Just wondering -- where did this come from? Was it an omission before?

Reply:
Oh wait, ignore that. I think I get it.

Reply:
Ah, without it, there's a noticeable disparity in the shadows between the top tutor card and the supplemental card. There should probably be a more cohesive way to address this though since I had to copy this over from the tutor card.